packages/rum-core/src/domain/assembly.spec.ts

```diff
@@ -430,10 +430,11 @@ describe('rum assembly', () => {
       expect(serverRumEvents[0].action).not.toBeDefined()
       serverRumEvents = []
 
+      const generatedRawRumActionEvent = createRawRumEvent(RumEventType.ACTION) as RawRumActionEvent
       notifyRawRumEvent(lifeCycle, {
-        rawRumEvent: createRawRumEvent(RumEventType.ACTION),
+        rawRumEvent: generatedRawRumActionEvent,
       })
-      expect((serverRumEvents[0] as RumActionEvent).action.id).not.toEqual('7890')
+      expect((serverRumEvents[0] as RumActionEvent).action.id).toEqual(generatedRawRumActionEvent.action.id)
```

Comment:
I made this change because I feared that `not.toEqual` was not specific enough now that it can be either an array or a string.

Reply:
Looks good